dts/bindings/interrupt-controller/arm,v8m-nvic.yaml

```diff
@@ -0,0 +1,32 @@
+---
+title: ARM Cortex M33 NVIC Interrupt Controller
```

Comment:
We shouldn't use Cortex M33 in the title/description (this is a mistake in the v6m & v7m yaml's as well).  Let just use some thing like V8M in this file and move this change to the first commit.

Reply:
@galak I would like also to fix the titles for the other .yaml files, to have some level of consistency.

Reply:
Added a third commit with the style updates on the v6m and v7m bindings.